### ISSUE TITLE
writeReport.DESeq2 Command Error: pandoc document conversion failed with error 83

### ISSUE THREAD

**Issue by MakeTheBrainHappy** [PF2-pasteur-fr/SARTools#81]
Dear SARTools Developers,

When implementing SARTools within the R Google Colaboratory I found everything working quite well except for the final command where I encountered the following error: https://u.cubeupload.com/MakeTheBrainHappy/ScreenShot20210410at.png 

Searching through public forums it seems to be related to an update to rmarkdown and the specific command utilized for citations. This also seems to have been addressed in the Galaxy version of SARTools: https://github.com/PF2-pasteur-fr/SARTools-Galaxy/issues/10 

I confirmed that the issue was with pandocs by setting it to false in a forked version: https://github.com/MakeTheBrainHappy/SARTools 

This fixed the problem well enough for the teaching session I was doing (we could kind of navigate through unparsed RTML); however, it would be great if there was a more permanent fix for when I release the notebook publicly. 

Thank you for your support!

Sincerely,
MakeTheBrainHappy

**Comment by hvaret** [comment]
Hi,
Thanks for the bug report. I have not worked on this piece of code for a long time, but did you try to install pandoc/pandoc-citeproc as specified on the home page of SARTools?
Best regards,
Hugo

**Comment by MakeTheBrainHappy** [comment]
Hi Hugo,

I confirmed that the linux system in Google Colaboratory has pandoc version 1.19.2.4~dfsg-1build4 installed. Since google colaboratory doesn't have a linux command line interface (when utilizing the R version) I needed to learn a little bit about R's system command to test versions. 

Once I added the following code (reverting to an older version of pandoc) the HTML report generated as expected. 

`a <- system("sudo wget https://github.com/jgm/pandoc/releases/download/1.15.1/pandoc-1.15.1-1-amd64.deb", intern = TRUE)
b <- system("sudo dpkg -i pandoc-1.15.1-1-amd64.deb", intern = TRUE)
a
b`

I assume therefore that the issue has to do with an update within the pandoc library. For my application, however, I have found a solution. Thank you for your help. 

Sincerely,
MakeTheBrainHappy
